tests/test_raster.py

```diff
@@ -527,5 +527,14 @@ def test_to_xyz_tiles(tmpdir, rioda_large):
         assert len(f.readlines()) == 1
 
 
-# def test_to_osm(tmpdir, dummy):
-#     dummy.raster.to_osm(
+def test_to_osm_tiles(tmpdir, rioda_large):
```

Comment:
It would be good to make this test a bit more precise, for instance check the bounding box of a specific output tile.

Reply:
Did that, maybe add a bit more later on.